src/backend/quarkus-management-api/src/main/java/com/nttdata/quarkus/management/api/model/api/Contact.java

```diff
@@ -0,0 +1,207 @@
+    return this;
+  }
+
+  /**
+   * Get contactId
+   * @return contactId
+  */
+  @ApiModelProperty(value = "")
+
+  @Valid
+
+  public BigDecimal getContactId() {
+    return contactId;
+  }
+
+  public void setContactId(BigDecimal contactId) {
+    this.contactId = contactId;
+  }
+
+  public Contact firstName(String firstName) {
+    this.firstName = firstName;
+    return this;
+  }
+
+  /**
+   * Get firstName
+   * @return firstName
+  */
+  @ApiModelProperty(value = "")
+
+
+  public String getFirstName() {
+    return firstName;
+  }
+
+  public void setFirstName(String firstName) {
+    this.firstName = firstName;
+  }
+
+  public Contact lastName(String lastName) {
+    this.lastName = lastName;
+    return this;
+  }
+
+  /**
+   * Get lastName
+   * @return lastName
+  */
+  @ApiModelProperty(value = "")
+
+
+  public String getLastName() {
+    return lastName;
+  }
+
+  public void setLastName(String lastName) {
+    this.lastName = lastName;
+  }
+
+  public Contact email(String email) {
+    this.email = email;
+    return this;
+  }
+
+  /**
+   * Get email
+   * @return email
+  */
+  @ApiModelProperty(value = "")
+
+
+  public String getEmail() {
+    return email;
+  }
+
+  public void setEmail(String email) {
+    this.email = email;
+  }
+
+  public Contact phoneNumber(String phoneNumber) {
+    this.phoneNumber = phoneNumber;
+    return this;
+  }
+
+  /**
+   * Get phoneNumber
+   * @return phoneNumber
+  */
+  @ApiModelProperty(value = "")
+
+
+  public String getPhoneNumber() {
+    return phoneNumber;
+  }
+
+  public void setPhoneNumber(String phoneNumber) {
+    this.phoneNumber = phoneNumber;
+  }
+
+  public Contact customerId(BigDecimal customerId) {
+    this.customerId = customerId;
+    return this;
+  }
+
+  /**
+   * Get customerId
+   * @return customerId
+  */
+  @ApiModelProperty(value = "")
+
+  @Valid
+
+  public BigDecimal getCustomerId() {
+    return customerId;
+  }
+
+  public void setCustomerId(BigDecimal customerId) {
+    this.customerId = customerId;
+  }
+
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    Contact contact = (Contact) o;
+    return Objects.equals(this.contactId, contact.contactId) &&
+        Objects.equals(this.firstName, contact.firstName) &&
+        Objects.equals(this.lastName, contact.lastName) &&
+        Objects.equals(this.email, contact.email) &&
+        Objects.equals(this.phoneNumber, contact.phoneNumber) &&
+        Objects.equals(this.customerId, contact.customerId);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(contactId, firstName, lastName, email, phoneNumber, customerId);
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("class Contact {\n");
+    
+    sb.append("    contactId: ").append(toIndentedString(contactId)).append("\n");
```

Comment:
why so many spaces in the string?

Reply:
Auto gen'd honestly I would never use the toString in this. There are better ways to do it.